### PR TITLE
Libexpat 2.2.0

### PR DESCRIPTION
**Pull request by catalinr**
Looks like all changes applied to 2.1.1 in wxW are included in 2.2.0.

### CODE REVIEW

File: src/expat/configure.ac [hunk 1]
```diff
@@ -45,18 +45,14 @@ dnl
...
+LIBREVISION=2  # with
+LIBAGE=6       # CMakeLists.txt!
 
 AC_CONFIG_HEADER(expat_config.h)
 
 sinclude(conftools/ac_c_bigendian_cross.m4)
 
-
-dnl Check for many C compilers (have to do this before libtool stuff)
-AC_BAKEFILE_PROG_CC
```
Comment (vadz) [inline src/expat/configure.ac]: Was this change (i.e. replacing this with `AC_PROG_CC` below) intentional?

**Review by vadz — APPROVED** [review]
Thanks, I didn't see anything else questionable except for this comment, if the CI builds pass, let's merge this.

Thanks again for your efforts!

File: src/expat/configure.ac (hunk 1 above)
Comment (catalinr) [inline src/expat/configure.ac]: Nope. It is part of changes in expat.
The first commit has the original expat code, and my only change was to redefine XML_STATIC in the same header as 2.1.1.

**Comment by vadz** [comment]
Thanks for the upgrade and for updating the docs!

**Pull request merged**